tests/func/test_get.py

```diff
@@ -36,6 +36,31 @@ def test_get_repo_dir(repo_dir, erepo):
     trees_equal(src, dst)
 
 
+def test_get_regular_file(erepo):
+    src = os.path.join(erepo.root_dir, "file")
+    dst = src + "_imported"
+    erepo.create(src, "hello")
+
+    Repo.get(erepo.root_dir, src, dst)
+
+    assert os.path.exists(dst)
+    assert os.path.isfile(dst)
+    assert filecmp.cmp(src, dst, shallow=False)
+
+
+def test_get_regular_dir(repo_dir, erepo):
```

Comment:
repo_dir is unnecessary here, can be removed

Reply:
@pared it is necessary, otherwise we will pollute repo root.

Reply:
@efiop I do not agree:
erepo already uses `repo_dir`, so temp test dir is already created, and we are back to it thanks to `os.chdir` inside erepo

Reply:
@pared Unless I'm missing something, if we take a look at erepo fixture https://github.com/iterative/dvc/blob/0.71.0/tests/conftest.py#L173 we can see that it returns back to the saved dir, which is our repository root. And then down below we do `Repo.get`, so the file will be downloaded to our repository root, which is bad.

Reply:
I disagree:
1. `erepo` uses `repo_dir` fixture, which gets created first and it creates temporary test dir and moves into it in `_pushd`
2. `erepo` creates `TestDvcGitFixture` which creates another temporary dir, but thanks to 1.
   its `_saved_dir` is actually our `repo_dir` root.
3. at the end of `erepo` creation, `chdir` moves back to `repo_dir` root

Reply:
Conclusion gentlemen?

Reply:
@daniholodovic The conclusion is that @pared tried it and it works, so he is right and removing is fine :) That being said, I'm a bit surprised by that behavior of `erepo`, don't remember if it was meant to do that. Seems like it creates a yet another test directory, which harmless but might not be what we need. Need to finally revisit our old unittest classes and new fixtures (I feel your pain @Suor 🙂).